control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -128,28 +129,6 @@ func ReconcileDeployment(deployment *appsv1.Deployment, auditWebhookRef *corev1.
 		InitContainers:               []corev1.Container{util.BuildContainer(oasTrustAnchorGenerator(), buildOASTrustAnchorGenerator(image))},
 		Containers: []corev1.Container{
 			util.BuildContainer(oasContainerMain(), buildOASContainerMain(image, strings.Split(etcdUrlData.Host, ":")[0], defaultOAPIPort, internalOAuthDisable)),
-			{
-				Name:            "audit-logs",
-				Image:           image,
-				ImagePullPolicy: corev1.PullIfNotPresent,
-				Command: []string{
-					"/usr/bin/tail",
-					"-c+1",
-					"-F",
-					fmt.Sprintf("%s/%s", volumeMounts.Path(oasContainerMain().Name, oasVolumeWorkLogs().Name), "audit.log"),
-				},
-				Resources: corev1.ResourceRequirements{
-					Requests: corev1.ResourceList{
-						corev1.ResourceCPU:    resource.MustParse("5m"),
-						corev1.ResourceMemory: resource.MustParse("10Mi"),
-					},
-				},
-				VolumeMounts: []corev1.VolumeMount{{
-					Name:      oasVolumeWorkLogs().Name,
-					MountPath: volumeMounts.Path(oasContainerMain().Name, oasVolumeWorkLogs().Name),
-				}},
-			},
-			util.BuildContainer(oasSocks5ProxyContainer(), buildOASSocks5ProxyContainer(socks5ProxyImage)),
```

Comment:
You need to leave `buildOASSocks5ProxyContainer`.  I think this is why verification tests are failing.

```
 control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go:235:6: func buildOASSocks5ProxyContainer is unused (U1000)
make: *** [Makefile:221: staticcheck] Error 1 
```

Reply:
@Joseph-Goergen ptal

Reply:
Should be fixed now. Sorry about the wait, was just distracted by other priorities on friday.